### ISSUE TITLE
Improve local development story

### ISSUE THREAD

**Issue by JustinBeckwith** [GoogleCloudPlatform/wombat-dressing-room#42]
I'm trying to follow the directions, and get the application running locally for development.  One of the things I'm getting tripped up on is the redirect Url for the GitHub OAuth application.  There's this blurb in the README:

> Note: the Authorization callback configured with the OAuth application should be the URL of the internal service, with the suffix /oauth/github.

For local setup - its unclear what that url is supposed to be.  I'm not sure if I need to configure this to be a `localhost:8080` address, or if I need to actually deploy the service just for the url redirect bits after login.  

I'm really just trying to slog my way through `local.env`.

**Comment by bcoe** [comment]
`config/local.env:`

```
NPM_OTP_SECRET=[secret]
NPM_TOKEN=[secret]
GITHUB_CLIENT_ID=[secret]
GITHUB_CLIENT_SECRET=[secret]
DATASTORE_PROJECT=my-datastore-remote-project
LOGIN_ENABLED=yes-this-is-a-login-server
LOGIN_URL=http://127.0.0.1:8080
REGISTRY_URL=http://127.0.01:8080
```

Authorization callback URL, for GitHub app:

```
http://127.0.0.1:8080/oauth/github
```

**Comment by JustinBeckwith** [comment]
Ok, so that one's on me 😆 I managed to point it towards `localhost:3000` out of habit. So ... thanks! As a note - I will go back through this thread when I'm done and submit a PR on the README and such. 

Now onto the next issue - got past login, and I'm looking at this url:
http://localhost:8080/_/manage-tokens

Getting an error like this on the page:
> error loading tokens. contact support with code e7181501-3ae5-4c34-ad4f-43700de5cbba

And getting this in the console:
```
querying tokens for <-----
error loading user tokens list JustinBeckwith error:
Error: 9 FAILED_PRECONDITION: no matching index found. recommended index is:
- kind: PublishKey
  properties:
  - name: username
  - name: created
```

Is it possible this is where we're trying to talk to datastore, and I'm missing something there?  I do have `DATASTORE_PROJECT` set, but I don't know how it's trying to authenticate.

**Comment by bcoe** [comment]
the first time you deploy, you need to also run `npm run create-indexes`, which can take a bit of time.

**Comment by JustinBeckwith** [comment]
Yeah I did that.  Turns out it was that I had `GOOGLE_APPLICATION_CREDENTIALS` set to a different project.  :/
